### PR TITLE
Add: Text-utilities

### PR DESCRIPTION
**Pull request by hussiantabha**
-headings
-small text
-gray text
-center text

### CODE REVIEW

File: docs.html [hunk 1]
```diff
@@ -404,6 +405,43 @@ <h1 class="display-title">Error Inputs</h1>
               </iframe>
             </div>
           </section>
+          <section class="display-code-content" id="text-utilities">
+            <div class="text-container">
+              <!--  -->
+              <h1 class="heading-bg">Heading BIG</h1>
+              <br>
+              <h2 class="heading-sm">Heading SMALL</h2>
+              <div>
```
Comment (rkapoor10) [inline docs.html:414]: you may add some more heading tags - like h3,h4....

File: docs.html [hunk 2]
```diff
@@ -404,6 +405,43 @@ <h1 class="display-title">Error Inputs</h1>
...
+                <iframe
+                  src="https://carbon.now.sh/embed?bg=rgba%28171%2C184%2C195%2C0%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253C%21--%2520Heading%2520Big%2520--%253E%250A%253Ch1%2520class%253D%2522heading-bg%2522%253EHeading%2520BIG%253C%252Fh1%253E%250A%253C%21--%2520Heading%2520Small%2520--%253E%250A%253Ch2%2520class%253D%2522heading-sm%2522%253EHeading%2520SMALL%253C%252Fh2%253E"
+                  style="width: 80%; height: 261px; border:0; transform: scale(1); overflow:hidden;"
+                  sandbox="allow-scripts allow-same-origin">
+                </iframe>
+              </div>
+              <h1 class="display-title">Gray text</h1>
+              <p class="gray-text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore, optio!</p>
```
Comment (rkapoor10) [inline docs.html:422]: you may add some paragraph text with different sizes

**Review by rkapoor10 — APPROVED** [review]
The code looks good; ready for merge.

**Review by paarthbajaj — APPROVED** [review]
Please use prettier, saying so because I am seeing bad indentation in iframes tag and then you are ready to merge.

File: docs.html (hunk 1 above)
Comment (hussiantabha) [reply docs.html:414]: Will do that

**Pull request merged**